code/game/machinery/_machinery.dm

```diff
@@ -97,8 +97,8 @@ Class Procs:
 		//0 = dont run the auto
 		//1 = run auto, use idle
 		//2 = run auto, use active
-	var/idle_power_usage = 0
-	var/active_power_usage = 0
+	var/idle_power_usage = NONE
+	var/active_power_usage = NONE
```

Comment:
ehhh, this is debatable. NONE is usually reserved for bitfields

Reply:
It's been used in a number of different spots for non-bitfields to rep 0 for a while now.